src/database-postgresql.cpp

```diff
@@ -0,0 +1,266 @@
+/*
+Copyright (C) 2016 Loic Blot <loic.blot@unix-experience.fr>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "config.h"
+
+#if USE_POSTGRESQL
+
+#include "database-postgresql.h"
+
+#include "log.h"
+#include "exceptions.h"
+#include "settings.h"
+
+/*
+ * IMPORTANT NOTICE
+ * Every conversion from non string to const char* values
+ * should be done by using an intermediate std::string
+ * buffer.
+ * If you don't use this intermediate, the conversion
+ * to std::string followed by conversion to const char*
+ * could have bad issues, refering to bad memory areas.
+ */
```

Comment:
If you have an idea how to fix this issue, i tried to do itos(string).c_str() and that failed, the postgresql connector doesn't like it